### PR TITLE
2.23 only has a 64bit version for mac

### PR DESCRIPTION
**Pull request by dampcake**
http://chromedriver.storage.googleapis.com/index.html?path=2.23/

### CODE REVIEW

**Comment by gauntface** [comment]
+1 to getting this PR in, but it would also be good if we could get the tests running on OS X as wall.

In travis you can do this with:

os:
  - linux
  - osx

**Comment by jak** [comment]
Can OSX tests go in another PR?

**Comment by chaseadamsio** [comment]
This is blocking a pretty significant number of users in Walmart Labs Frontend Engineering team, it would be super helpful if this could be merged & versioned soon!

File: install.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ if (platform === 'linux') {
     platform += '32'
   }
 } else if (platform === 'darwin') {
-  platform = 'mac32'
+  platform = 'mac64'
```
Comment (chaseadamsio) [inline install.js]: I know it's not super common for people to be running 32-bit OSX today, but should this also have a conditional check for `process.arch == x64`?

**Comment by dampcake** [comment]
chaseadamsio, there isn't a 32 bit build :(

**Comment by chaseadamsio** [comment]
@dampcake yeah, I've seen that for this version, but it still feels like the right thing to do would be to:

- have the `32` build in the `platform` if someone is on that arch
- if the binaries not available for anything, surface a message for that and make it chromedriver's problem by suggesting to go and ask for it. 😎 

That's my thought at least.

**Comment by chaseadamsio** [comment]
if we'd had that for 2.22, we'd have a 64-bit Darwin version: http://chromedriver.storage.googleapis.com/index.html?path=2.22/

**Comment by chaseadamsio** [comment]
`ChromeDriver installation failed undefined` is the response people are getting for when there's no such chromedriver from `err.stack`, maybe at the very least there's a way to catch that the error is that there's no url for that and that people should check the url requested.

I think setting it to `64` without checking first can cause the same results people had when we didn't check for `32` which is that they have no idea what went wrong and `32` was technically wrong for most architectures anyways.

**Comment by giggio** [comment]
It seems that there is no mac 32 anymore in `2.23`, just checkout http://chromedriver.storage.googleapis.com/index.html?path=2.23/. Until 2.22 there was only mac32: http://chromedriver.storage.googleapis.com/index.html?path=2.22/.
So, yes this is a bug. I will verify it.

File: install.js (hunk 1 above)
Comment (giggio) [inline install.js]: @chaseadamsio there is no mac32 anymore. Then what, we just fail the installation?

**Comment by giggio** [comment]
@dampcake @gauntface I integrated the PR and added osx as another OS for travis, but it is failing the build:
https://travis-ci.org/giggio/node-chromedriver/jobs/153057355
I don't have a Mac to verify, would you please check? The error is:

````
/Users/travis/build.sh: line 45: timeout: command not found
The command "npm install && timeout 10 ./bin/chromedriver || test $? -eq 124" exited with 1.
Done. Your build exited with 1.
````

If it is ok, I will publish it to NPM, but first I need to either remove osx from Travis (bad, as we may have problems again in the future), or understand why it is failing (better).

**Comment by gauntface** [comment]
Remove it for now (seems important enough to fix right now) and I'll try to reproduce locally.

**Comment by giggio** [comment]
I agree, I am removing it and will publish next. The command that is failing is:

````
npm install && timeout 10 ./bin/chromedriver || test $? -eq 12
````

**Comment by giggio** [comment]
Following @chaseadamsio idea, if the user is running Mac 32 bits we will now fail the installation.

**Comment by christophercotton** [comment]
@giggio node.js doesn't list 32 bit installers for OSX.

**Comment by jak** [comment]
Apple machines have also been 64 bit for about 10 years now

**Comment by chaseadamsio** [comment]
I'm pretty sure there are some intel core duos that are 32-bit from early 2013 still out there.

**Comment by giggio** [comment]
Working version published to NPM as version `2.23.1`

**Pull request merged**